.github/autolabeler.yml

```diff
@@ -73,7 +73,7 @@ io:  ["sdks/java/io/*"]
 # Runners
 "runners": ["runners/*"]
 "apex": ["runners/apex/*"]
-"core": ["runners/core-construction-java/*","runners/core-java/*"]
+"core": ["runners/core-construction-java/*", "runners/core-java/*"]
```

Comment:
Because you removed the prefix, I'm not so happy with the resulting core label. You also have an SDK core, it seems confusing. This has nothing todo with this PR though, so LGTM

Reply:
Again it is a question of tradeoffs, we don't have an 'sdks' category so the argument of the second core does not hold (even if we have sdks/core).
Alex I did not do this to annoy you, but with valid arguments on readability and usability, prove me that the prefix version is simpler and better and I don't hesitate to change to it, but I am still unconvinced of what we win with the extra prefix vs the extra noise and reduced readability it brings specially when the prefixes are already covered by the general categories.